tests/data/spec-08-13.canonical

```diff
@@ -1,7 +1,7 @@
 %YAML 1.1
 ---
 !!map {
-  ? !!str "foo"
+  ? !!str "foo :"
```

Comment:
It passed, but the key was parsed as "foo". Note that pyyaml and libyaml behave differently in this edge case, libyaml rejects any colon in a flow context without a space afterwards, while pyyaml allows ',[]{}' after a colon as well: https://github.com/yaml/pyyaml/blob/master/lib/yaml/scanner.py#L1301